internal/k8s/status.go

```diff
@@ -259,13 +261,22 @@ func getExternalServiceAddress(svc *api_v1.Service) []string {
 func (su *statusUpdater) SaveStatusFromExternalStatus(externalStatusAddress string) {
 	su.externalStatusAddress = externalStatusAddress
 	if externalStatusAddress == "" {
-		// if external-status-address was removed from configMap, fall back on
-		// external service if it exists
+		// if external-status-address was removed from configMap
+
+		// fall back on external service if it exists
 		if len(su.externalServiceAddresses) > 0 {
 			su.saveStatus(su.externalServiceAddresses)
 			su.externalEndpoints = su.generateExternalEndpointsFromStatus(su.status)
 			return
 		}
+
+		// fall back on NginxCisConnector if it exists
+		if len(su.bigIPAddress) > 0 {
```

Comment:
Any reason why not?:
```suggestion
		if su.bigIPAddress != "" {
```

While equivalent, I found the len check misleading. I thought `su.bigIPAddress` must be a slice or something.